cssEditor.js

```diff
@@ -0,0 +1,49 @@
+
+function setupEditor() {
+    window.editor = ace.edit("editor");
+    editor.setTheme("ace/theme/tomorrow_night_blue");
+    editor.getSession().setMode("ace/mode/html");
+    editor.setValue(`<!DOCTYPE html>
+    <html>
+        <head>
+
+        <style>
+
+        </style>
+        </head>
+        
+        <body>
+
+        </body>
+
+    </html>`,1); //1 = moves cursor to end
+
+    editor.getSession().on('change', function() {
+        update();
+    });
+
+    editor.focus();
+    
+    
+    editor.setOptions({
+        fontSize: "11pt",
+        showLineNumbers: false,
+        showGutter: false,
+        vScrollBarAlwaysVisible:true,
+        enableBasicAutocompletion: false, enableLiveAutocompletion: false
+    });
+
+    editor.setShowPrintMargin(false);
+    editor.setBehavioursEnabled(false);
+}
+
+function update() {
+    let idoc = document.getElementById("iframe").contentWindow.document;
```

Comment:
i love that you're using an iframe. security! you still would need to do a bit more to ensure things are fully secure, though. good stuff!